modules/apps/redirect/redirect-web/src/main/resources/META-INF/resources/js/RedirectManagementToolbarPropsTransformer.js

```diff
@@ -12,28 +12,33 @@
  * details.
  */
 
-import {DefaultEventHandler} from 'frontend-js-web';
-import {Config} from 'metal-state';
+import {postForm} from 'frontend-js-web';
 
-class RedirectManagementToolbarDefaultEventHandler extends DefaultEventHandler {
-	deleteSelectedRedirectEntries() {
-		const form = this.one('#fm');
+export default function propsTransformer({
+	additionalProps: {deleteRedirectEntriesURL},
+	portletNamespace,
+	...otherProps
+}) {
+	const deleteSelectedRedirectEntries = () => {
+		const form = document.getElementById(`${portletNamespace}fm`);
```

Comment:
We can use ``document.[`${portletNamespace}fm`]`` here.

Reply:
> We can use document.[`${portletNamespace}fm`] here.

I think this is a somewhat obscure API. I think we had a conversation about this a while ago. I'm not sure if we reached any conclusion, but my instinct would be to avoid it. It always surprises me when I see it in usage.

@wincent, do you recall what we discuss about this?

Reply:
Just to add a little bit more information on this. You can refer to the [document object](https://html.spec.whatwg.org/#the-document-object) part of the spec.

This API has on particular big gotcha:

> The Document interface supports named properties. The supported property names of a Document object document at any moment consist of the following, in tree order according to the element that contributed them, ignoring later duplicates, and with values from id attributes coming before values from name attributes when the same element contributes both:
>
>- the value of the name content attribute for all exposed embed, form, iframe, img, and exposed object elements that have a **non-empty name** content attribute and are in a document tree with document as their root;
>...

So, for `<form id="foo" name="foo">` then `document.foo === document.getElementById('foo')` but for `<form id="foo" name="bar">` then `document.foo !== document.getElementById('foo')

Reply:
@jbalsas got it, we'll avoid using it from now on.